Functions.Templates/Templates/SqlOutputBinding-JavaScript/index.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const testRow = {
+        "column1": column1Value,
+        "column2": "column2Value"
+    };
+
+    context.bindings.results = req["body"]?.rows ?? JSON.stringify(testRow);
+
+    return {
+        status: 201,
+        body: context.bindings.results
```

Comment:
`mimetype:"application/json"`

Reply:
I don't have this in our samples, return the mimetype as well?

Reply:
Yes, we need to always be doing this. mimetype tells the browser how to render the body - so when we send json like this we need to tell it that to ensure things handle it correctly.